lib/rtcpeerconnection/safari.js

```diff
@@ -40,6 +40,13 @@ function SafariRTCPeerConnection(configuration) {
       value: null,
       writable: true
     },
+    // NOTE(syerrapragada): There is small time frame when remote offer is applied
+    // but _pendingRemoteOffer is not cleared causing an issue in OrderedTrackMatcher
+    // in twilio-video.js.
+    _isPendingRemoteOfferApplied: {
```

Comment:
@syerrapragada Instead of a new boolean property, can we just set `_pendingRemoteOffer` to `null` in a "track" event listener like so:
```js
this._peerConnection.addEventListener('track', () => this._pendingRemoteOffer = null);
```
This way, its set to null before `remoteDescription` is accessed in `_handleTrackEvent` in `PeerConnectionV2`.

Reply:
I will create another PR while close this one. Thanks Manjesh